.gitlab/llnl-ci-spack.yml

```diff
@@ -52,6 +52,11 @@ stages:
     - spack config add concretizer:unify:when_possible
     - spack config add concretizer:reuse:false
     - spack config add config:db_lock_timeout:300
+    - spack config add packages:ant:buildable:false
+    - spack config add packages:ant:externals:spec:ant@1.10.5
+    # - spack config add packages:ant:externals:prefix:\/usr # FIXME/NOTE: uncomment once spack PR: https://github.com/spack/spack/pull/40976 is merged
```

Comment:
I think it will be with double quotes
`spack config add packages:ant:externals:prefix:"/usr"`